bitnami/mongodb-sharded/Chart.yaml

```
@@ -38,4 +38,4 @@ maintainers:
 name: mongodb-sharded
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/mongodb-sharded
-version: 9.3.6
+version: 9.3.7
```

According to semver we are adding a new feature, so we should bump the minor version:

```suggestion
version: 9.4.0
```